SKILL.tsv

```diff
@@ -5155,7 +5155,7 @@ SKILL_20150918_005154	$Applies one new deadly poison to your weapon for the next
 SKILL_20150918_005155	$Immune to Lv. 1 abnormal status
 SKILL_20150918_005156	$Increased Magic Defense.
 SKILL_20150918_005157	Shield Bash-Mix stat
-SKILL_20150918_005158	Weaknesss to blow
+SKILL_20150918_005158	Vulnerable to attack
```

Comment:
There are several other describing weakness to a specific attack style. Would you like weakness or vulnerability? Similar 'Immune to' and 'Immunity against'